tests/test_server.py

```diff
@@ -1024,8 +1025,9 @@ def test_delete_running_backup(
             server.delete_backup(backup_info_empty)
             out, err = capsys.readouterr()
             assert (
-                "Cannot delete a running backup (%s %s)"
-                % (server.config.name, backup_info_started.backup_id)
+                "Another action is in progress on the %s backup"
```

Comment:
"Another action is in progress for the backup %s"

Reply:
Thanks, updated